cmd/newallocation.go

```diff
@@ -151,12 +151,21 @@ var newallocationCmd = &cobra.Command{
 			}
 		}
 
-		var expire time.Duration
-		if expire, err = flags.GetDuration("expire"); err != nil {
-			log.Fatal("invalid 'expire' flag: ", err)
+		config, err := sdk.GetStorageSCConfig()
```

Comment:
we can completely remove this flag for now, and simply hardcode 1 time-unit in smart contract. in the future we can have 1,2,3.. time-units for an allocation when we buy it